testproject/Assets/Tests/Runtime/MultiprocessRuntime/ThreeDText.cs

```diff
@@ -0,0 +1,75 @@
+using UnityEngine;
+using Unity.Netcode;
+#if UNITY_UNET_PRESENT
+using Unity.Netcode.Transports.UNET;
+#endif
+
+namespace Unity.Netcode.MultiprocessRuntimeTests
+{
+    public class ThreeDText : MonoBehaviour
+    {
+        public bool IsTestCoordinatorActiveAndEnabled = false;
+        public string CommandLineArguments = "";
+        private long m_UpdateCounter;
+        private bool m_HasFired;
+        // Start is called before the first frame update
+        public void Start()
+        {
+            Debug.Log("ThreeDText - Start");
+            m_HasFired = false;
+            m_UpdateCounter = 0;
+            var jsonTextFile = Resources.Load<TextAsset>("Text/multiprocess_tests");
+            Debug.Log(jsonTextFile);
+
+            var t = GetComponent<TextMesh>();
+            t.text = "On Start";
+            CommandLineArguments = System.Environment.CommandLine;
+
+            string[] args = System.Environment.GetCommandLineArgs();
+            foreach (var arg in args)
+            {
+                Debug.Log(arg);
+                CommandLineArguments += " " + arg;
+            }
+            Debug.Log($"CommandLineArguments {CommandLineArguments}");
+        }
+
+        // Update is called once per frame
+        public void Update()
+        {
+            m_UpdateCounter++;
+            var testCoordinator = TestCoordinator.Instance;
+            if (testCoordinator == null)
+            {
+                return;
+            }
+
+            var transport = NetworkManager.Singleton != null ? NetworkManager.Singleton.NetworkConfig.NetworkTransport : null;
+            var transportString = "";
+            if (transport == null)
+            {
+                transportString = "null";
+            }
+            else
+            {
+                transportString = transport.ToString();
+            }
+
+            var t = GetComponent<TextMesh>();
+
+            if (IsTestCoordinatorActiveAndEnabled != testCoordinator.isActiveAndEnabled ||
+                !m_HasFired ||
+                m_UpdateCounter % 100 == 0)
+            {
+                m_HasFired = true;
+                t.text = $"On Update -\ntestCoordinator.isActiveAndEnabled:{testCoordinator.isActiveAndEnabled}\n"+
```

Comment:
that's a lot of string manipulation to do in something that would be used for performance testing. We might want to make sure this is disabled during perf tests.
Even with the m_UpdateCounter check, unless we're fixing our target update rate, this has the potential to run quite often.

Reply:
that would include debug logs as well

Reply:
Ah yes, good point. I'll change this to update only on state change. Let me take another crack at this.

Reply:
even on state change, this could introduce perf issues for that particular frame. Could we just disable completely this component when doing perf tests?

Reply:
@SamuelBellomo I just pushed an update to this, let me know if the reduction in number of updates is better.

Reply:
Ok, I can check if we are in a perftest and just skip this entirely.